webapp/param_parser.py

```diff
@@ -0,0 +1,19 @@
+from flask import request
+from webapp.dataclass import AssetSearchParams
+
+
+def parse_asset_search_params() -> AssetSearchParams:
+    """
+    Parse request arguments and return AssetSearchParams object with defaults.
+    """
+    return AssetSearchParams(
+        tag=request.args.get("tag", "").strip(),
+        asset_type=request.args.get("asset_type", "").strip(),
+        product_types=request.args.getlist("product_types") or [],
+        author_email=request.args.get("author_email", "").strip(),
+        title=request.args.get("title", "").strip(),
```

Comment:
Here you use `title`

Reply:
Oh, alright so the thing is that is the Figma has disparity in it, it says it `asset name` during creation and `Title` during search.

I kept like that for sake of UI and frontend but we can definitely make it consistent.
So what shall we keep?

Reply:
![image](https://github.com/user-attachments/assets/82742224-82c2-4755-a7bd-5aa36c8e5ec6)

![Uploading image.png…]()

Reply:
I would make it 'title' everywhere